parm/config/config.base.emc.dyn

```diff
@@ -117,8 +117,11 @@ export ATARDIR="@ATARDIR@"
 
 # Commonly defined parameters in JJOBS
 export envir=${envir:-"prod"}
-export NET="gfs"
-export RUN=${RUN:-${CDUMP:-"gfs"}}
+export NET="gfs"  # NET is defined in the job-card (ecf)
+export RUN=${RUN:-${CDUMP:-"gfs"}}  # RUN is defined in the job-card (ecf); CDUMP is used at EMC as a RUN proxy
+# TODO: determine where is RUN actually used in the workflow other than here
+# TODO: is it possible to replace all instances of ${CDUMP} to ${RUN} to be
+#       consistent w/ EE2?
```

Comment:
With the caveat that GEFS is still using the v15-based g-w, GEFS is definitely using these differently. It often sets `$CDUMP` to the `$RUNMEM` (`gec00`, `gep01`, `gep02` ... `geaer`), but `$NET` and `$RUN` remain `gefs`. Now, this may no longer be necessary if GEFS output is reorganized to look like EnKF, but GEFS running these scripts as an external system running on top of g-w will need to be considered.

```
Orion-login-3 GEFS/>egrep '\WCDUMP=' -R
scripts/exgefs_forecast.sh:export CDUMP=$RUNMEM
parm/gefs_wave_prep.parm:export CDUMP=$RUNMEM
Orion-login-3 GEFS/>egrep '\WRUN=' -R
rocoto/bin/gefs_dev/JGEFS_CHEM_POST_CLEANUP:export RUN=${RUN:-gefs}
rocoto/bin/gefs_dev/jobs/JGEFS_ATMOS_AVG_GEMPAK_VGF:export RUN=${RUN:-gefs}
rocoto/bin/sh/JGEFS_WAFS:export RUN=${RUN:-gefs}
rocoto/bin/sh/JGEFS_POST_TRACK:export RUN=${RUN:-gefs}
rocoto/bin/sh/JGEFS_POST_TRACK_GFS:export RUN=${RUN:-gefs}
rocoto/bin/sh/JGEFS_POST_GENESIS:export RUN=${RUN:-gefs}
rocoto/bin/sh/JGEFS_CQPF:export RUN=cqpf
rocoto/bin/sh/JGEFS_RF_PREP:export RUN=${RUN:-gefs}
rocoto/bin/sh/JGEFS_EXTRACTVARS:export RUN=${RUN:-gefs}
rocoto/parm/gefs_config:export RUN=${RUN:-gefs}
rocoto/dev/lsf/wcoss2/lsf_common.sh:export RUN=${RUN:-gefs}
jobs/JGEFS_ATMOS_INIT_RECENTER:export RUN=${RUN:-gefs}
jobs/JGEFS_FCST_POST_MANAGER:export RUN=${RUN:-gefs}
jobs/JGEFS_ATMOS_POST_CLEANUP:export RUN=${RUN:-gefs}
jobs/JGEFS_WAVE_GEMPAK:export RUN=${RUN:-gefs}
jobs/JGEFS_ATMOS_POSTSND:export RUN=${RUN:-gefs}
jobs/JGEFS_WAVE_INIT:export RUN=${RUN:-gefs}
jobs/JGEFS_ATMOS_GETCFSSST:export RUN=${RUN:-gefs}
jobs/JGEFS_WAVE_POST:export RUN=${RUN:-gefs}
jobs/JGEFS_ATMOS_GEMPAK:export RUN=${RUN:-gefs}
jobs/JGEFS_CHEM_INIT:export RUN=${RUN:-gefs}
jobs/JGEFS_WAVE_PREP:export RUN=${RUN:-gefs}
jobs/JGEFS_ATMOS_ENSAVG_NEMSIO:export RUN=${RUN:-gefs}
jobs/JGEFS_ATMOS_ENSSTAT:export RUN=${RUN:-gefs}
jobs/JGEFS_ATMOS_GEMPAK_META:export RUN=${RUN:-gefs}
jobs/JGEFS_ATMOS_AWIPS:export RUN=${RUN:-gefs}
jobs/JGEFS_ATMOS_POST:export RUN=${RUN:-gefs}
jobs/JGEFS_ATMOS_AVGSPR_GEMPAK_META:export RUN=${RUN:-gefs}
jobs/JGEFS_FORECAST:export RUN=${RUN:-gefs}
jobs/JGEFS_ATMOS_ENSPOST:export RUN=${RUN:-gefs}
jobs/JGEFS_ATMOS_PRDGEN:export RUN=${RUN:-gefs}
jobs/JGEFS_WAVE_STAT:export RUN=${RUN:-gefs}
jobs/JGEFS_ATMOS_PREP:export RUN=${RUN:-gefs}
jobs/JGEFS_CHEM_PREP_EMISSIONS:export RUN=${RUN:-gefs}
```

Reply:
Noted.
Though that means `GEFS` is misusing the reservation of `CDUMP` and its interpretation is different from that of `GFS`